specification.md

```diff
@@ -691,10 +713,13 @@ KD/4Axmo9DISib5/7A6uczJxQG2Bcrdj++vQqK2succ=
 
 All Parties **MUST** make best efforts to not throttle during normal operation.
 
-Controllers **MAY** use a sub-processor to manage their OpenGDPR request infrastructure
+Controllers **MAY** use a sub-processor to manage their OpenCompliance request infrastructure
 to ensure requests are distributed reliably, security signatures are verified and
 thorough logs are kept.
 
+## 10.1 Migration from OpenGDPR
```

Comment:
Maybe it would be better for reading to have Version `1.0` and `2.0` in this `specification.md` file or separate `specification_v1.md` and `specification_v2.md` files.

Reply:
I was planning to tag a github branch with V1 and leave it for posterity with a link in the README so that it's clear how to read the v1 version.

Reply:
I agree this works too.